language-plutus-core/stdlib/Language/PlutusCore/StdLib/Data/Integer.hs

```diff
@@ -0,0 +1,30 @@
+-- | Functions related to @integer@.
+
+{-# LANGUAGE OverloadedStrings #-}
+module Language.PlutusCore.StdLib.Data.Integer
+    ( getBuiltinSuccInteger
+    ) where
+
+import           Language.PlutusCore.Constant (makeDynamicBuiltinInt)
+import           Language.PlutusCore.MkPlc
+import           Language.PlutusCore.Name
+import           Language.PlutusCore.Quote
+import           Language.PlutusCore.Type
+
+-- |  @foldInteger@ as a PLC term.
```

Comment:
Wrong name

Reply:
Thanks. Also it's `0` in the comment rather than `1`.